src/main/java/core/basesyntax/strategy/operation/BalanceOperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy.operation;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public int getQuantity(int prev, int value) {
```

Comment:
Consider adding a more descriptive javadoc for the `getQuantity` method to explain its purpose, especially since it's implementing an interface method. This would improve code readability and maintainability.